src/README.md

```diff
@@ -1,15 +1,95 @@
-The docs are built on GitHub Actions and run the ``.ci_scripts/update_docs`` script.
+The docs are built on GitHub Actions and run the `.ci_scripts/update_docs` script.
 
 To build the docs locally, follow the steps mentioned below:
- 1.  Clone the repository into your local machine.
- 2.  Go into the main folder and run the following commands.  
- 3. ``conda env create -f ./.ci_scripts/environment.yml``
- 4. ``conda activate conda-forge-docs``
- 5. ``cd newsfeed && pip install --no-deps .``
- 6. ``cd ../src``
- 7. ``make html``
 
-Documentation update PRs should only change the files in ``src``. 
+1.  Clone the repository into your local machine.
+2.  Go into the main folder and run the following commands.
 
-Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
+```
+conda env create -f ./.ci_scripts/environment.yml
+```
 
+```
+conda activate conda-forge-docs
+```
+
+```
+cd newsfeed && pip install --no-deps .
+```
+
+```
+cd ../src
+```
+
+```
+make html
+```
+
+# Contributor NumFocus Code of Conduct
```

Comment:
I don't think we need NumFocus's Code of Conduct in here. conda-forge would need to have its own. We don't want to just copy theirs.

Reply:
Ooh, looks like you already did that. Cool.

Reply:
Thank you for your feedback @kathatherine
 if they are any other thing I need to add to the readme, please let me know, so I wrap this PR up and work on another issue.

 I really want to constantly contribute to this project.